src/core/components/collapsible/collapsible.test.tsx

```diff
@@ -0,0 +1,105 @@
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
We've a package installed that allows better user event simulation. You can see it used in the DialogHeader tests as well as the DatalistFilter ones.

```ts
import { userEvent } from @testing-library/user-event
```